packages/app/src/pages/PoolPage/PoolsPreview.tsx

```diff
@@ -0,0 +1,96 @@
+import { BiListUl } from "react-icons/bi";
+import { useNavigate } from "react-router-dom";
+
+import { Accordion } from "~/components/Accordion";
+import { Button } from "~/components/Button";
+import { PreviewTable, PreviewItem } from "~/components/PreviewTable";
+import { Spinner } from "~/components/Spinner";
+import { TokenIcon } from "~/components/TokenIcon";
+import { usePoolInfo } from "~/hooks/usePoolInfo";
+import { calculateRatio } from "~/lib/asset";
+import { COIN_DAI, COIN_ETH } from "~/lib/constants";
```

Comment:
This ingo about the two tokens should be provided from the LP token and then we can query it.

Meta code:
- LP Token -> ETH/DAI
- CoinsMeta.filter(c => c.symbol === ETH | DAI);

Reply:
✅